doc/reports/spikes/metrics-what-we-want-spike.md

```diff
@@ -0,0 +1,18 @@
+# Monitoring Performance - What should we monitor?
+## Introduction
+This spike report follows on from work and discussions in the OAT meeting to identify areas where we want to be monitoring the system. This outlines what I see as an idealised monitoring setup - there may be blockers to overcome to achieve some of this. I've also provided guidance in the document to how I see the monitoring being provided and used.
+## Production metrics
+All of these metrics should be radiated on monitors
+### Performance
```

Comment:
Might be worth looking at these metric as SLIs (Service Level Indicators) - would then need to establish what the SLOs (Service Level Objectives) for each actually are (though gathering baselines is the first step)

Reply:
I support this, but it's beyond the scope of this spike (and is a job for OAT)